java-client/session/src/main/java/io/deephaven/client/impl/FetchedObject.java

```diff
@@ -5,28 +5,35 @@
 
 import com.google.protobuf.ByteString;
 
+import java.io.Closeable;
 import java.io.IOException;
 import java.io.OutputStream;
 import java.util.Collections;
 import java.util.List;
 import java.util.Objects;
+import java.util.stream.Collectors;
 
-public final class FetchedObject {
+/**
+ * The results from fetching an object.
+ *
+ * @see Session#fetchObject(HasTypedTicket)
+ */
+public final class FetchedObject implements Closeable {
```

Comment:
In light of the other new types, I'm not sure that this class makes sense by itself - shouldn't this implement ServerObject or at least HasTypedTicket/etc? This type functionally is the result of fetching a ServerObject using the legacy/deprecated FetchObject call - perhaps we deprecate it and avoid adding new  functionality to it, encourage users to move to CustomObject, which should fully replace it?

Reply:
To confirm, it is definitely deprecated for removal, and we want to remove the extra junk in the impl that exists only to bridge the gap for the unary impl. Ryan reviewed and allowed some of the ugliness with the understanding that it was going away "Soon".

What is the difference between FetchedObject and a ServerObject that is only fetchable? With FetchObject removed, is it possible to create a FetchedObject, either now or in the future? The plugin can't specify that one vs the other is the only way to fetch an object either.